backend/src/models/renter.ts

```diff
@@ -0,0 +1,17 @@
+import { InferSchemaType, Schema, model } from "mongoose";
+
+const renterSchema = new Schema(
+  {
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    contactInfo: { type: String, required: true },
+    program: { type: String, required: false },
+  },
+  // Mongoose will automatically create "createdAt" and "updatedAt" properties
+  // and update them accordingly when the document is saved/updated.
```

Comment:
Would you like to keep or remove the comments?

Reply:
Removed the comments